app/src/main/java/com/rentminder/dao/IHouseholdDAO.kt

```diff
@@ -6,5 +6,5 @@ import retrofit2.http.GET
 
 interface IHouseholdDAO {
     @GET("")
-    fun getAllCountries() : Call<ArrayList<Household>>
+    fun getAllHouseholds() : Call<ArrayList<Household>>
```

Comment:
Good suggestion, but this same suggestion appeared in a previously submitted code review.
When that occurs, only the first gets credit.